test/command_callback/test_erb_command_callback.vader

```diff
@@ -0,0 +1,21 @@
+Before:
+  runtime ale_linters/eruby/erb.vim
+  call ale#test#SetDirectory('/testplugin/test/command_callback')
+
+After:
+  call ale#linter#Reset()
+  call ale#test#RestoreDirectory()
+
+Execute(Executable should not contain any filter code by default):
+  call ale#test#SetFilename('../ruby_fixtures/not_a_rails_app/file.rb')
+
+  AssertEqual
+  \ 'erb -P -x %t | ruby -c',
+  \ ale_linters#eruby#erb#GetCommand(bufnr(''))
+
+Execute(Executable should filter invalid eRuby when inside a Rails project):
+  call ale#test#SetFilename('../ruby_fixtures/valid_rails_app/app/views/my_great_view.html.erb')
+
+  AssertEqual
+  \ 'ruby -rerb -e "puts ERB.new(File.read(%t, encoding: \"BINARY\").gsub(\"<%=\",\"<%\"), nil, \"-\").src" | ruby -c',
```

Comment:
You'll have to use `ale#Escape` in the test for the parameter with the suggestion above. The string will differ between Linux and Windows.